src_c/mask.c

```diff
@@ -1555,27 +1555,42 @@ static PyTypeObject pgMask_Type = {
 /*mask module methods*/
 
 static PyObject *
-Mask(PyObject *self, PyObject *args)
+Mask(PyObject *self, PyObject *args, PyObject *kwargs)
 {
     bitmask_t *mask;
     int w, h;
+    int fill = 0; /* Default is false. */
     pgMaskObject *maskobj;
-    if (!PyArg_ParseTuple(args, "(ii)", &w, &h))
+    char *keywords[] = {"", "fill", NULL};
```

Comment:
I think `size` could be named as well. Guess it's for backward compatibility?

Reply:
Thanks for the comment.

I was just going to leave it as an unnamed positional-only parameter, but giving it a name is a good idea I've made the change.